bench/src/bin/bulk.rs

```diff
@@ -155,3 +186,30 @@ async fn handle_client_stream(
 
     Ok((upload_result, download_result))
 }
+
+struct ClientStats {
+    upload_stats: Stats,
+    download_stats: Stats,
+}
+
+impl ClientStats {
+    pub fn new() -> Self {
+        Self {
+            upload_stats: Stats::default(),
+            download_stats: Stats::default(),
+        }
+    }
```

Comment:
Derive `Default` for `ClientStats` instead?

Reply:
makes sense. updated